examples/shapes/shapes_math_sine_cosine.c

```diff
@@ -0,0 +1,180 @@
+/*******************************************************************************************
+ *
+ *   raylib [shapes] example - math sine cosine
+ *
+ *   Port of the p5.js "sine and cosine" demo to raylib + raygui
+ *   Sources: https://p5js.org/examples/angles-and-motion-sine-cosine/
+ *
+ *   Demonstrates uniform circular motion and the correspondence between
+ *   the unit circle and sine / cosine graphs. Right panel contains a
+ *   play/pause toggle implemented with raygui.
+ * 
+ *   Example originally created with raylib 5.6-dev, last time updated with raylib 5.6-dev
+ *
+ *   Copyright (c) 2025 Olivier (@oqu)
+ *
+ ********************************************************************************************/
+
+#include "raylib.h"
+
+// Required for GUI controls
+#define RAYGUI_IMPLEMENTATION
+#include "raygui.h"
+
+#include <math.h>
+
+int main(void)
+{
+    // Window initialization
+    const int screenWidth = 800;
+    const int screenHeight = 450;
+
+    InitWindow(screenWidth, screenHeight, "raylib [shapes] example - math sine cosine");
+    const char* playText = "Play";
+    const char* pauseText = "Pause";
+
+    // Scene layout (mimic p5 example positioning)
+    const float circleX = 200.0f;
+    const float circleY = 150.0f;
+    const float circleRadius = 75.0f;
+
+    const float graphX = 50.0f;
+    const float graphY = 300.0f;
+    const float graphAmplitude = 50.0f;
+    const float graphPeriod = 300.0f; // width of the graph in pixels
+
+    // Animation / UI state
+    bool playing = true;     // play / pause toggle
+    int frameCountLocal = 0; // local frame counter (used when playing)
```

Comment:
Please, review raylib code conventions: https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md

Comments should start with capital letter